app/controllers/forum/categories/category/threads/thread/posts/index.js

```diff
@@ -55,8 +54,12 @@ export default class PostsIndexController extends Controller {
 
   @action
   async addQuote(q) {
-    this.set('newContent', `${this.newContent}${q} \n\n`);
-
+    if (!this.newContent?.endsWith('\n')) {
```

Comment:
Unnecessary chaining :p, confused me because now it looks like `this.newContent` could be undefined/null and this conditional will pass then because !undefined/!null == true